libs/ui/src/lib/button/button.component.stories.ts

```diff
@@ -86,3 +86,19 @@ const TemplateTextButtonWithIcon: Story<ButtonComponent> = (
 });
 
 export const TextButtonWithIcon = TemplateTextButtonWithIcon.bind({});
+
+const TemplateSmallButtonWithIcon: Story<ButtonComponent> = (
+  args: ButtonComponent
+) => ({
+  props: args,
+  /* template */
+  template: `
+    <cvi-ng-button [size]="size">
+      <cvi-ng-labeled-icon name="exit_to_app" svgClass="cvi-button__white-icon" appearance="secondary" iconPosition="after" verticalAlignment="center" [iconHeight]="24">
+        {{ content }}
+      </cvi-ng-labeled-icon>
```

Comment:
With the change suggested for the CSS of `cvi-button`, no need to apply custom `svgClass` class in the example. However, `cvi-ng-labeled-icon` overrides the fill set in `cvi-ng-button`. 

Instead of using `cvi-ng-labeled-icon`, we could replace it directly with `cvi-ng-track` and `cvi-ng-icon`: 

```suggestion
<cvi-ng-track [gap]="4">
  {{ content }}
  <cvi-ng-icon name="exit_to_app" verticalAlignment="center" [height]="14"></cvi-ng-icon>
</cvi-ng-track>
```

Reply:
This does not seem to work. Somehow when inside track cvi-ng-icon component does not account for its height attribute, so it looks off-center 
![image](https://github.com/e-gov/cvi/assets/22071435/5ad329b8-dfa3-499e-9f5d-7ce9f377941a)

Would it be ok to keep the cvi-ng-labeled icon since the fill inside #{$base}--appearance-primary has higher specificity due to targeting path?

Reply:
Hey @helemand, let me investigate a bit, let's see if there's a good solution that would work for all similar cases.

Reply:
The specificity issue was caused by a `fill=none` attribute on `svg` tag of the `exit_to_app` icon, overriding any inherited fill set in CSS. It is [highly recommended](https://e-gov.github.io/cvi/?path=/docs/icons-how-to-contribute--page) to remove those from all the SVG icons.

Reply:
> This does not seem to work. Somehow when inside track cvi-ng-icon component does not account for its height attribute, so it looks off-center

Yes, you are right. One of the ways to fight this it is to add `display: flex` or `display: inline-flex` to the icon wrapper... This means though that the whole setup needs to be encapsulated in a component (since all the components should work in such a way as not to require any extra CSS). 

This is already fixed, it seems.